include/cantera/numerics/Jacobian.h

```diff
@@ -0,0 +1,100 @@
+//! @file MultiJac.h
```

Comment:
Pretty much `MultiJac.h`, modified for a single ODE system and switched from BandMatrix to DenseMatrix. Considering moving this functionality into `Newton` once solver structure is finalized.

Reply:
Removed dependence on this class in paulblum/cantera@d2789f9, deleted in paulblum/cantera@8bd7385.